server/package.json

```diff
@@ -1,16 +1,30 @@
 {
-  "name": "server",
+  "name": "dashboard",
   "version": "1.0.0",
   "description": "",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "server": "nodemon server.js",
+    "client": "npm start --prefix client",
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "client-install": "npm install --prefix client",
+    "disc": "node discord.js",
+    "youtube": "node youtube.js"
```

Comment:
I think we should keep `start`, `server`, `disc`, and `youtube`. The rest won't work anymore

Reply:
fair